javascript2/week1/script.js

```diff
@@ -0,0 +1,123 @@
+let recipeObject = {
+  id: 1,
+  title: "Vegetable Biryani",
+  picture_url: "./images/veg_biryani.jpeg",
+  ingredients: [
+    { NAME: "bay leaf", AMOUNT: "2" },
+    { NAME: "cinnamon", AMOUNT: "2 inch" },
+    { NAME: "cloves", AMOUNT: "6" },
+    { NAME: "star anise", AMOUNT: "2" },
+    { NAME: "basmati rice", AMOUNT: "2 cups" },
+    { NAME: "green peas", AMOUNT: "1/4 cup" },
+    { NAME: "carrot chopped", AMOUNT: "1/4 cup" },
+    { NAME: "green beans chopped", AMOUNT: "1/4 cup" },
+    { NAME: "potatoes chopped", AMOUNT: "1/4 cup" },
+    { NAME: "onions chopped", AMOUNT: "2 medium sized" },
+    { NAME: "tomatoes chopped", AMOUNT: "2 small" },
+    { NAME: "chopped fresh mint", AMOUNT: "4 teaspoons" },
+    { NAME: "ginger garlic paste", AMOUNT: "2 teaspoon" },
+    { NAME: "red chili powder", AMOUNT: "1 teaspoon" },
+    { NAME: "turmeric powder", AMOUNT: "0.5 teaspoon" },
+    { NAME: "garam masala", AMOUNT: "1.5 teaspoon" },
+    { NAME: "oil", AMOUNT: "3-4 tablespoons" },
+    { NAME: "water ", AMOUNT: "3.5 cups" },
+    { NAME: "salt ", AMOUNT: "as required" },
+  ],
+  description:
+    "Wash basmati rice in clean water twice and soak it for 15-20 minutes. Heat oil in a thick bottom pan/ pressure cooker, add bay leaf, star anise, cloves, and cinnamon. Saute it for 1 minute. Add onions, saute for 2 to 3 mins till the onions turn translucent. Next add tomatoes, ginger-garlic paste and saute until the raw smell has gone off. Add all the chopped vegetables and fry for about 2 minutes. Next add salt, mint leaves, red chilli powder, turmeric, and garam masala. Mix all of these and fry it for 2 to 3 mins. Next, add water and bring it to a boil in a high flame. When it starts boiling, open the lid and add soaked basmati rice. Mix well and close the lid and allow it to boil again. Cook in low flame for 15 to 20 minutes. Now veg biryani is ready.",
+};
+
+function displayRecipe(recipe) {
```

Comment:
Good stuff with this function.

If you want to get a little more advanced, you could consider making the `recipe-section` div into an unordered list (`<ul>`), then display each individual recipe inside of a list item (`<li>`).

At the moment its not a big deal since you only have the one starting recipe. But it's good to represent things with the most semantically-aligned HTML element we can, since that helps screen readers, search engine crawlers, and other tools determine how your site is structured. Eventually the site will have a list of recipes rather than just one, so the markup should reflect that.